buku

```diff
@@ -2002,8 +2006,6 @@ class BukuDb:
 
```

Comment:
…Come to think of it – should this be actually printed out when not in CLI mode?